vespalib/src/vespa/vespalib/net/server_socket.h

```diff
@@ -20,7 +20,7 @@ class ServerSocket
 
     void cleanup();
 public:
-    ServerSocket() : _handle(), _path() {}
+    ServerSocket() : _handle(), _path(), _blocking(false), _shutdown() {}
```

Comment:
In a weak moment I imaged that `std::atomic` did implicit initialization 🙂  Will fix

Reply:
Apparently atomics get value-initialized as of C++20 (https://en.cppreference.com/w/cpp/atomic/atomic/atomic)

> Value-initializes the underlying object (i.e. with `T()`). The initialization is not atomic. Use of the default constructor is ill-formed if [`std::is_default_constructible_v<T>`](http://en.cppreference.com/w/cpp/types/is_default_constructible) is false.

The More You Know™ 🦄. Still should prefer to explicitly initialize the stuff

Reply:
I thought it would be initialized as well... I was more commenting the difference in approach when initializing two booleans (explicit vs. default-value).